shared/logutil/logutil.go

```diff
@@ -23,3 +25,18 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+func CountdownToGenesis(genesisTime time.Time, secondsCount int) {
```

Comment:
Please add a godoc comment for this public method

Reply:
Okay, is there a template to follow?